apps/web-connect/src/features/checkout/components/ActionSection.tsx

```diff
@@ -48,7 +47,7 @@ export function ActionSection(): JSX.Element {
      */
     const getTokenButtonText = useCallback(() => {
         if (mintWithEth.isPending || mintWithXai.isPending || approve.isPending) return "WAITING FOR CONFIRMATION..";
-        if (chain?.id !== 42161) return "Please Switch to Arbitrum One";
+        if (chain?.id !== MAINNET_ID && chain?.id !== TESTNET_ID) return "Please Switch to Arbitrum";
```

Comment:
These `chain?.id !== MAINNET_ID && chain?.id !== TESTNET_ID` need to only allow `TESTNET_ID` when we are in a develop/testing environment. In production we only want to allow `MAINNET_ID`. Please update at all places :)

Reply:
Added a new util function in core and updated all references.